---
title: Publications
permalink: /publications/
---


1. G. Gessain et al., “Trem2-expressing
multinucleated giant macrophages are a biomarker of good prognosis in
head and neck squamous cell carcinoma,” *Cancer Discovery*, Sep. 2024,
doi:
[10.1158/2159-8290.CD-24-0018](https://doi.org/10.1158/2159-8290.CD-24-0018).
1. D. Zyss, A. Sharma, S. A. Ribeiro, C. E. Repellin, O. Lai, T. Walter,
and A. Fehri, “Contrastive learning for cell division detection and
tracking in live cell imaging data.” BioRXiv, Aug. 2024, doi:
[10.1101/2024.08.16.608296](https://doi.org/10.1101/2024.08.16.608296).
1. T. Defard, H. Laporte, M. Ayan, J. Soulier, S. Curras-Alonso, C. Weber,
F. Massip, J.-A. Londoño-Vallejo, C. Fouillade, F. Mueller, and T.
Walter, “A point cloud segmentation framework for image-based spatial
transcriptomics,” *Communications Biology*, vol. 7, no. 1, p. 823, Jul.
2024, doi:
[10.1038/s42003-024-06480-3](https://doi.org/10.1038/s42003-024-06480-3).
1. N. Captier, M. Lerousseau, F. Orlhac, N. Hovhannisyan-Baghdasarian, M.
Luporsi, E. Woff, S. Lagha, P. Salamoun Feghali, C. Lonjou, C.
Beaulaton, H. Salmon, T. Walter, I. Buvat, N. Girard, and E. Barillot,
“Integration of clinical, pathological, radiological, and transcriptomic
data improves the prediction of first-line immunotherapy outcome in
metastatic non-small cell lung cancer.” medRXiv, preprint, Jun. 2024, doi:
[10.1101/2024.06.27.24309583](https://doi.org/10.1101/2024.06.27.24309583).
1. C. A. Jahangir et al., 
“Image-based multiplex immune profiling of cancer tissues: Translational
implications. A report of the International Immuno-oncology Biomarker
Working Group on Breast Cancer,” *The Journal of Pathology*, vol. 262,
no. 3, pp. 271–288, Mar. 2024, doi:
[10.1002/path.6238](https://doi.org/10.1002/path.6238).
1. X. Devos, J.-B. Fiche, M. Bardou, O. Messina, C. Houbron, J. Gurgo, M.
Schaeffer, M. Götz, T. Walter, F. Mueller, and M. Nollmann, “<span
class="nocase">pyHiM</span>: A new open-source, multi-platform software
package for spatial genomics based on multiplexed DNA-FISH imaging,”
*Genome Biology*, vol. 25, no. 1, p. 47, Feb. 2024, doi:
[10.1186/s13059-024-03178-x](https://doi.org/10.1186/s13059-024-03178-x).
1. A. Beaufrère, T. Lazard, R. Nicolle, G. Lubuela, J. Augustin, M.
Albuquerque, B. Pichon, C. Pignolet, V. Priori, N. Théou-Anton, M.
Lesurtel, M. Bouattour, K. Mondet, J. Cros, J. Calderaro, T. Walter, and
V. Paradis, “Self-supervised learning to predict intrahepatic
cholangiocarcinoma transcriptomic classes on routine histology,” BioRXiv, Jan.
2024, doi:
[10.1101/2024.01.15.575652](https://doi.org/10.1101/2024.01.15.575652).
1. T. Lazard, M. Lerousseau, S. Gardrat, A. Vincent-Salomon, M.-H. Stern,
M. Rodrigues, E. Decencière, and T. Walter, “Democratizing computational
pathology: Optimized Whole Slide Image representations for The Cancer
Genome Atlas,” BioRXiv, Preprint, Dec. 2023. doi:
[10.1101/2023.12.04.569894](https://doi.org/10.1101/2023.12.04.569894).
1. T. Defard, H. Laporte, M. Ayan, S. Juliette, S. Curras-Alonso, C. Weber,
F. Massip, J.-A. Londoño-Vallejo, C. Fouillade, F. Mueller, and T.
Walter, “A point cloud segmentation framework for image-based spatial
transcriptomics,” BioRXiv, Preprint, Dec. 2023. doi:
[10.1101/2023.12.01.569528](https://doi.org/10.1101/2023.12.01.569528).
1. Lazard, Tristan, Bataillon, Guillaume, Walter, Thomas, and Vincent
Salomon, Anne, “Cancer du sein - utilisation de l’intelligence
artificielle pour prédire le statut tumoral relatif à la recombinaison
homologue,” *Med Sci (Paris)*, vol. 39, no. 12, pp. 926–928, 2023, doi:
[10.1051/medsci/2023169](https://doi.org/10.1051/medsci/2023169).
1. M. Lubrano *et al.*, “Diagnosis with confidence: Deep learning for
reliable classification of laryngeal dysplasia,” *Histopathology*, p.
his.15067, Oct. 2023, doi:
[10.1111/his.15067](https://doi.org/10.1111/his.15067).
1. D. B. Page *et al.*, “Spatial analyses of immune cell infiltration in
cancer: Current methods and future directions: A report of the
International Immuno‐Oncology Biomarker Working Group on Breast Cancer,”
*The Journal of Pathology*, vol. 260, no. 5, pp. 514–532, Aug. 2023,
doi: [10.1002/path.6165](https://doi.org/10.1002/path.6165).
1. J. Thagaard *et al.*, “Pitfalls in machine learning‐based assessment of
tumor‐infiltrating lymphocytes in breast cancer: A report of the
International Immuno‐Oncology Biomarker Working Group on Breast Cancer,”
*The Journal of Pathology*, vol. 260, no. 5, pp. 498–513, Aug. 2023,
doi: [10.1002/path.6155](https://doi.org/10.1002/path.6155).
1. M. Lubrano, Y. Bellahsen-Harrar, R. Fick, C. Badoual, and T. Walter,
“Simple and Eﬃcient Conﬁdence Score for Grading Whole Slide Images” in
*Proceedings of Machine Learning Research*, Jul. 2023, vol. 58, pp.
1–19.[link](https://openreview.net/pdf?id=DA1hOTvcMWa)
1. D. Zyss, S. A. Ribeiro, M. J. C. Ludlam, T. Walter, and A. Fehri, “Cell
Segmentation in Images Without Structural Fluorescent Labels,” *Biol.
Imaging*, pp. 1–18, Jul. 2023, doi:
[10.1017/S2633903X23000168](https://doi.org/10.1017/S2633903X23000168).
1. Y. Bellahsen-Harrar *et al.*, “AI-Augmented Pathology for Head and Neck
Squamous Lesions Improves Non-HN Pathologist Agreement to Expert Level,”
Pathology, preprint, Jul. 2023. doi:
[10.1101/2023.07.23.23292962](https://doi.org/10.1101/2023.07.23.23292962).
1. T. Lazard, M. Lerousseau, E. Decencière, and T. Walter, “Giga-SSL: <span
class="nocase">Self-supervised</span> learning for gigapixel images,” in
*Proceedings of the IEEE/CVF conference on computer vision and pattern
recognition (CVPR) workshops*, Jun. 2023, pp. 4304–4313. [link](https://openaccess.thecvf.com/content/CVPR2023W/CVMI/papers/Lazard_Giga-SSL_Self-Supervised_Learning_for_Gigapixel_Images_CVPRW_2023_paper.pdf)
1. S. Curras-Alonso *et al.*, “An interactive murine single-cell atlas of
the lung responses to radiation injury,” *Nat Commun*, vol. 14, no. 1,
p. 2445, Apr. 2023, doi:
[10.1038/s41467-023-38134-z](https://doi.org/10.1038/s41467-023-38134-z).
1. M. Lubrano, Y. Bellahsen-Harrar, R. Fick, C. Badoual, and T. Walter,
“Simple and efficient confidence score for grading whole slide images.”
arXiv, Mar. 08, 2023. doi:
[10.48550/arXiv.2303.04604](http://arxiv.org/abs/2303.04604)
1. T. Bonte, M. Philbert, E. Coleno, E. Bertrand, A. Imbert, and T. Walter,
“Learning with Minimal Effort: Leveraging in Silico Labeling for Cell
and Nucleus Segmentation,” in *Computer Vision – ECCV 2022 Workshops*,
2023, pp. 423–436, doi:
[10.1007/978-3-031-25069-9\_28](https://doi.org/10.1007/978-3-031-25069-9_28).
1. A. Imbert, F. Mueller, and T. Walter, “PointFISH: Learning Point Cloud
Representations for RNA Localization Patterns,” in *Computer Vision –
ECCV 2022 Workshops*, 2023, pp. 487–502, doi:
[10.1007/978-3-031-25069-9\_32](https://doi.org/10.1007/978-3-031-25069-9_32).
1. M. Lubrano *et al.*, “Automatic Grading of Cervical Biopsies
by Combining Full and <span class="nocase">Self-supervision</span>,” in
*Computer Vision – ECCV 2022 Workshops*, 2023, pp. 408–423, doi:
[10.1007/978-3-031-25082-8\_27](https://doi.org/10.1007/978-3-031-25082-8_27).
1. A. Imbert, F. Mueller, and T. Walter, “PointFISH – learning point cloud
representations for RNA localization patterns.” arXiv, Feb. 21, 2023,
Accessed: Mar. 11, 2023. doi: [10.48550/arXiv.2302.10923](http://arxiv.org/abs/2302.10923)
1. T. Bonte, M. Philbert, E. Coleno, E. Bertrand, A. Imbert, and T. Walter,
“Learning with minimal effort: Leveraging in silico labeling for cell
and nucleus segmentation.” arXiv, Jan. 10, 2023. doi: [10.48550/arXiv.2301.03914](http://arxiv.org/abs/2301.03914)
1. T. Lazard, M. Lerousseau, E. Decencière, and T. Walter, “Giga-SSL:
Self-Supervised Learning for Gigapixel Images.” arXiv, Dec. 2022, doi: [10.48550/arXiv.2212.03273](http://arxiv.org/abs/2212.03273).
1. M. Lubrano, Y. Bellahsen-Harrar, S. Berlemont, T. Walter, and C.
Badoual, “Diagnosis with Confidence: Deep Learning for Reliable
Classification of Squamous Lesions of the Upper Aerodigestive Tract,”
BioRxiv, preprint, Dec. 2022. doi:
[10.1101/2022.12.21.521392](https://doi.org/10.1101/2022.12.21.521392).
1. T. Lazard *et al.*, “Deep learning identifies morphological patterns of
homologous recombination deficiency in luminal breast cancers from whole
slide images,” *Cell Reports Medicine*, p. 100872, Dec. 2022, doi:
[10.1016/j.xcrm.2022.100872](https://doi.org/10.1016/j.xcrm.2022.100872).
1. A. Safieddine *et al.*, “HT-<span class="nocase">smFISH</span>: A
cost-effective and flexible workflow for high-throughput single-molecule
RNA imaging,” *Nature Protocols*, Oct. 2022, doi:
[10.1038/s41596-022-00750-2](https://doi.org/10.1038/s41596-022-00750-2).
1. A. Imbert *et al.*, “<span class="nocase">FISH-quant</span> v2: A
scalable and modular tool for <span class="nocase">smFISH</span> image
analysis,” *RNA*, p. rna.079073.121, Mar. 2022, doi:
[10.1261/rna.079073.121](https://doi.org/10.1261/rna.079073.121).
1. P. Naylor *et al.*, “Prediction of Treatment Response in Triple Negative
Breast Cancer From Whole Slide Images,” *Frontiers in Signal
Processing*, vol. 2, p. 851809, Jun. 2022, doi:
[10.3389/frsip.2022.851809](https://doi.org/10.3389/frsip.2022.851809).
1. M. Lubrano di Scandalea *et al.*, “Automatic grading of cervical biopsies
by combining full and self-supervision,” *bioRxiv*, p.
2022.01.14.476330, Jan. 2022, doi:
[10.1101/2022.01.14.476330](https://doi.org/10.1101/2022.01.14.476330).
1. T. Lazard *et al.*, “Deep learning identifies new morphological patterns
of homologous recombination deficiency in luminal breast cancers from
whole slide images,” *bioRxiv*, 2021, doi:
[10.1101/2021.09.10.459734](https://doi.org/10.1101/2021.09.10.459734)
1. A. Imbert *et al.*, “FISH-quant v2: A scalable and modular analysis tool
for smFISH image analysis,” *bioRxiv*, 2021, doi:
[10.1101/2021.07.20.453024](https://doi.org/10.1101/2021.07.20.453024).
1. A. Safieddine *et al.*, “A choreography of centrosomal <span
class="nocase">mRNAs</span> reveals a conserved localization mechanism
involving active polysome transport,” *Nature Communications*, vol. 12,
no. 1, p. 1352, Mar. 2021, doi:
[10.1038/s41467-021-21585-7](https://doi.org/10.1038/s41467-021-21585-7).
1. S. Curras-Alonso, J. Soulier, T. Walter, F. Mueller, A. Londoño-Vallejo,
and C. Fouillade, “Spatial transcriptomics for respiratory research and
medicine,” *Eur Respir J*, p. 2004314, Jan. 2021, doi:
[10.1183/13993003.04314-2020](https://doi.org/10.1183/13993003.04314-2020).
1. X. Pichon *et al.*, “The kinesin KIF1C transports APC-dependent mRNAs to
cell protrusions,” *RNA*, 2020, doi:[10.1261/rna.078576.120](https://doi.org/10.1261/rna.078576.120).
1. M. Balluet *et al.*, “Neural network fast-classifies biological images
using features selected after their random-forests-importance to power
smart microscopy.” 2020, \[Online\]. Available: <http://arxiv.org/abs/2012.10331>.
1. R. Chouaib *et al.*, “A Dual Protein-<span class="nocase">mRNA
Localization Screen Reveals Compartmentalized Translation</span> and
Widespread Co-translational RNA Targeting,” *Developmental Cell*, vol.
54, no. 6, pp. 773–791.e5, Sep. 2020, doi:
[10.1016/j.devcel.2020.07.010](https://doi.org/10.1016/j.devcel.2020.07.010).
2. T. Walter, “<span class="nocase">Bioimage Informatics for
Phenomics</span>,” Habilitation à diriger des recherches, Sorbonne
Université, 2020,[tel-02981391](https://hal.inria.fr/ENSMP_DEP_MS/tel-02981391v1).
3. J. Boyd, Z. Gouveia, F. Perez, and T. Walter, “Experimentally-Generated
Ground Truth for Detecting Cell Types in an Image-Based Immunotherapy
Screen,” in *2020 IEEE 17th International Symposium on Biomedical
Imaging (ISBI)*, Apr. 2020, pp. 886–890, doi:
[10.1109/ISBI45749.2020.9098696](https://doi.org/10.1109/ISBI45749.2020.9098696).
4. J. C. Boyd, A. Pinheiro, E. D. Nery, F. Reyal, and T. Walter,
“Domain-invariant features for mechanism of action prediction in a
multi-cell-line drug screen,” *Bioinformatics*, Oct. 2019, doi:
[10.1093/bioinformatics/btz774](https://doi.org/10.1093/bioinformatics/btz774).
5. M. Durand *et al.*, “Human lymphoid organ <span
class="nocase">cDC2</span> and macrophages play complementary roles in T
follicular helper responses,” *The Journal of Experimental Medicine*,
vol. 216, no. 7, pp. 1561 LP–1581, Jul. 2019, doi:
[10.1084/jem.20181994](https://doi.org/10.1084/jem.20181994).
6. R. Dubois *et al.*, “A deep learning approach to identify <span
class="nocase">mRNA</span> localization patterns,” in *IEEE 16th
International Symposium on Biomedical Imaging (ISBI 2019)*, 2019, pp.
1386–1390, doi:
[10.1109/ISBI.2019.8759235](https://doi.org/10.1109/ISBI.2019.8759235).
7. I. Kuperstein and E. Barillot, Eds., *Computational Systems Biology
Approaches in Cancer Research*. Chapman and Hall/CRC, 2019.
8. P. Naylor, J. Boyd, M. Laé, F. Reyal, and T. Walter, “Predicting
Residual Cancer Burden In A Triple Negative Breast Cancer Cohort,” in
*2019 IEEE 16th International Symposium on Biomedical Imaging (ISBI
2019)*, 2019, pp. 933–937, doi:
[10.1109/ISBI.2019.8759205](https://doi.org/10.1109/ISBI.2019.8759205).
9. P. Naylor, M. La, F. Reyal, and T. Walter, “Segmentation of Nuclei in
Histopathology Images by deep regression of the distance map,” *IEEE
Transactions on Medical Imaging*, vol. 38, no. 2, pp. 448–459, 2019,
doi: [10.1109/TMI.2018.2865709](https://doi.org/10.1109/TMI.2018.2865709).
11. J. Boyd, A. Pinhiero, E. D. Nery, F. Reyal, and T. Walter, “Analysing
double-strand breaks in cultured cells for drug screening applications
by causal inference,” in *International Symposium on Biomedical Imaging
(ISBI): From Nano to Macro*, 2018, vol. 2018–April, pp. 445–448, doi:
[10.1109/ISBI.2018.8363612](https://doi.org/10.1109/ISBI.2018.8363612).
12. A. Samacoits *et al.*, “A computational framework to study sub-cellular
RNA localization,” *Nature Communications*, vol. 9, no. 1, p. 4584,
2018, doi: [10.1038/s41467-018-06868-w](https://doi.org/10.1038/s41467-018-06868-w).
13. E. Bernard, Y. Jiao, E. Scornet, V. Stoven, T. Walter, and J.-P. Vert,
“Kernel multitask regression for toxicogenetics,” *Molecular
Informatics*, vol. 36, Jan. 2017, \[Online\]. Available: <http://biorxiv.org/content/early/2017/08/01/171298.abstract>.
14. P. Naylor, M. Lae, F. Reyal, and T. Walter, “Nuclei Segmentation in
Histopathology Images Using Deep Neural Networks,” *2017 IEEE 14th
International Symposium on Biomedical Imaging (ISBI 2017)*, pp. IEEE;
EMB; IEEE Signal Proc Soc, 2017, doi:
[10.1109/ISBI.2017.7950669](https://doi.org/10.1109/ISBI.2017.7950669).
15. M. Isokane *et al.*, “ARHGEF17 is an essential spindle assembly
checkpoint factor that targets Mps1 to kinetochores,” *The Journal of
Cell Biology*, vol. 212, no. 6, pp. 647–659, Mar. 2016, doi:
[10.1083/jcb.201408089](https://doi.org/10.1083/jcb.201408089).
16. V. Machairas, T. Baldeweck, T. Walter, and Etienne Décencière, “New
General Features Based on Superpixels for image Segmentation Learning,”
in *International Symposium on Biomedical Imaging (ISBI): From Nano to
Macro*, 2016, pp. (accepted for publication).
17. N. Tsanov *et al.*, “SmiFISH and FISH-quant - A flexible single RNA
detection approach with super-resolution capability,” *Nucleic Acids
Research*, vol. 44, no. 22, 2016, doi:
[10.1093/nar/gkw784](https://doi.org/10.1093/nar/gkw784).
18. F. Eduati *et al.*, “Prediction of human population responses to toxic
compounds by a collaborative competition,” *Nature Biotechnology*, vol.
33, p. 933, Aug. 2015, \[Online\]. Available: <https://doi.org/10.1038/nbt.3299>.
20. V. Machairas, E. Decencière, and T. Walter, “Spatial Repulsion Between
Markers Improves Watershed Performance BT - Mathematical Morphology and
Its Applications to Signal and Image Processing,” 2015, pp. 194–202.
21. V. Machairas, M. Faessel, D. Cárdenas-peña, T. Chabardes, T. Walter, and
E. Decencière, “Waterpixels,” *IEEE Transactions on Image Processing*,
vol. 24, no. 11, pp. 3707–3716, 2015.
22. A. Schoenauer Sebag, S. Plancade, C. Raulet-Tomkiewicz, R. Barouki,
J.-P. Vert, and T. Walter, “A generic methodological framework for
studying single cell motility in high-throughput time-lapse data,”
*Bioinformatics*, vol. 31, no. 12, pp. i320–i328, 2015, doi:
[10.1093/bioinformatics/btv225](https://doi.org/10.1093/bioinformatics/btv225).
23. A. Schoenauer Sebag, S. Plancade, C. Raulet-Tomkiewicz, R. Barouki,
J.-P. Vert, and T. Walter, “Infering an ontology of single cell motions
from high-throughput microscopy data,” in *Proceedings of the 12th IEEE
International Symposium on Biomedical Imaging (ISBI): From nano to
macro*, 2015, pp. 160–163.
24. J. Tegha-Dunghu *et al.*, “MAP1S controls microtubule stability
throughout the cell cycle in human cells,” *Journal of Cell Science*,
vol. 127, no. 23, pp. 5007 LP–5013, Dec. 2014, doi:
[10.1242/jcs.136457](https://doi.org/10.1242/jcs.136457).
25. J. C. Costello *et al.*, “A community effort to assess and improve drug
sensitivity prediction algorithms.” *Nature biotechnology*, vol. 32, no.
12, pp. 1–103, 2014, doi:
[10.1038/nbt.2877](https://doi.org/10.1038/nbt.2877).
26. V. Graml *et al.*, “A Genomic Multiprocess Survey of Machineries that
control and Link Cell Shape, Microtubule Organization, and Cell-Cycle
Progression,” *Developmental Cell*, vol. 31, no. 2, pp. 227–239, 2014,
doi:
[10.1016/j.devcel.2014.09.005](https://doi.org/10.1016/j.devcel.2014.09.005).
27. J.-K. Hériché, J. G. Lees, I. Morilla, T. Walter, and B. Petrova,
“Integration of biological data by kernels on graph nodes allows
prediction of new genes involved in mitotic chromosome condensation,”
*Molecular Biology of the cell*, vol. 25, pp. 2522–2536, 2014, doi:
[10.1091/mbc.E13-04-0221](https://doi.org/10.1091/mbc.E13-04-0221).
28. V. Machairas, E. Decencière, and T. Walter, “Waterpixels: Superpixels
based on the watershed transformation,” in *International Conference on
Image Processing (ICIP)*, 2014, pp. 4343–4347.
29. M. Veta *et al.*, “Assessment of algorithms for mitosis detection in
breast cancer histopathology images,” *Medical Image Analysis*, pp.
1–23, 2014.
30. G. Pau, T. Walter, B. Neumann, J. Hériché, J. Ellenberg, and W. Huber,
“Dynamical modelling of phenotypes in a genome-wide RNAi live-cell
imaging assay,” *BMC bioinformatics*, vol. 14, no. 308, pp. 1–10, 2013,
doi:
[10.1186/1471-2105-14-308](https://doi.org/10.1186/1471-2105-14-308).
31. M. Mall *et al.*, “Mitotic lamin disassembly is triggered by
lipid-mediated signaling.” *The Journal of cell biology*, vol. 198, no.
6, pp. 981–90, Sep. 2012, doi:
[10.1083/jcb.201205103](https://doi.org/10.1083/jcb.201205103).
32. C. Conrad *et al.*, “Micropilot: Automation of fluorescence
microscopy-based imaging for systems biology.” *Nature methods*, vol. 8,
no. 3, pp. 246–9, Mar. 2011, doi:
[10.1038/nmeth.1558](https://doi.org/10.1038/nmeth.1558).
33. M. Held *et al.*, “CellCognition: Time-resolved phenotype annotation in
high-throughput live cell imaging.” *Nature methods*, vol. 7, no. 9, pp.
747–54, Sep. 2010, doi:
[10.1038/nmeth.1486](https://doi.org/10.1038/nmeth.1486).
34. B. Dupas *et al.*, “Evaluation of automated fundus photograph analysis
algorithms for detecting microaneurysms, haemorrhages and exudates, and
of a computer-assisted diagnostic system for grading diabetic
retinopathy.” *Diabetes & metabolism*, vol. 36, no. 3, pp. 213–220, Jun.
2010, doi:
[10.1016/j.diabet.2010.01.002](https://doi.org/10.1016/j.diabet.2010.01.002).
35. T. Walter *et al.*, “Phenotypic profiling of the human genome by
time-lapse microscopy reveals cell division genes.” *Nature*, vol. 464,
no. 7289, pp. 721–7, Apr. 2010, doi:
[10.1038/nature08869](https://doi.org/10.1038/nature08869).
36. T. Walter *et al.*, “Automatic identification and clustering of
chromosome phenotypes in a genome wide RNAi screen by time-lapse
imaging.” *Journal of structural biology*, vol. 170, no. 1, pp. 1–9,
Apr. 2010, doi:
[10.1016/j.jsb.2009.10.004](https://doi.org/10.1016/j.jsb.2009.10.004).
37. S. I. O. Donoghue *et al.*, “Visualizing biological data — now and in
the future,” *Nature Methods*, vol. 7, pp. S2–S4, 2010, doi:
[10.1038/nmeth0310-S2](https://doi.org/10.1038/nmeth0310-S2).
38. S. Terjung, T. Walter, A. Seitz, B. Neumann, R. Pepperkok, and J.
Ellenberg, “High-throughput microscopy using live mammalian cells.” in
*Live Cell Imaging: A Laboratory Manual*, R. D. Goldman, J. R. Swedlow,
and D. L. Spector, Eds. Cold Spring Harbor Laboratory Press, Cold Spring
Harbor, 2010.
39. T. Walter *et al.*, “Visualization of image data from cells to
organisms,” *Nature Methods*, vol. 7, no. 3, pp. S26–S41, 2010, doi:
[10.1038/NmEtH.1431](https://doi.org/10.1038/NmEtH.1431).
40. J. Tegha-Dunghu *et al.*, “EML3 is a nuclear microtubule-binding protein
required for the correct alignment of chromosomes in metaphase,”
*Journal of Cell Science*, vol. 121, no. 10, pp. 1718 LP–1726, May 2008,
doi: [10.1242/jcs.019174](https://doi.org/10.1242/jcs.019174).
41. T. Walter *et al.*, “A genome wide RNAi screen by time lapse microscopy
in order to identify mitotic genes - Computational aspects and
challenges,” *2008 5th IEEE International Symposium on Biomedical
Imaging: From Nano to Macro, Proceedings, ISBI*, pp. 328–331, 2008, doi:
[10.1109/ISBI.2008.4540999](https://doi.org/10.1109/ISBI.2008.4540999).
42. E. Denion, J.-R. Ordonez, J.-C. Klein, A. Glacet-Bernard, T. Walter, and
G. Caputo, “Redistribution of the neurosensory retina in inferior
limited macular translocation: An evaluation using image registration.”
*Graefe’s archive for clinical and experimental ophthalmology = Albrecht
von Graefes Archiv fur klinische und experimentelle Ophthalmologie*,
vol. 245, no. 3, pp. 437–442, Mar. 2007, doi:
[10.1007/s00417-006-0408-1](https://doi.org/10.1007/s00417-006-0408-1).
43. G. Sihn *et al.*, “Anti-angiogenic properties of myo-inositol
trispyrophosphate in ovo and growth reduction of implanted glioma.”
*FEBS letters*, vol. 581, no. 5, pp. 962–6, Mar. 2007, doi:
[10.1016/j.febslet.2007.01.079](https://doi.org/10.1016/j.febslet.2007.01.079).
44. H. Erfle *et al.*, “Reverse transfection on cell arrays for high content
screening microscopy,” *Nature Protocols*, vol. 2, no. 2, pp. 392–399,
Feb. 2007, doi:
[10.1038/nprot.2006.483](https://doi.org/10.1038/nprot.2006.483).
45. T. Walter, P. Massin, A. Erginay, R. Ordonez, C. Jeulin, and J.-C.
Klein, “Automatic detection of microaneurysms in color fundus images.”
*Medical Image Analysis*, vol. 11, no. 6, pp. 555–66, 2007.
46. T. Walter, R. Ordonez, and J.-C. Klein, “A morphological approach for
skeleton filtering with reconstruction of the relevant branches,”
presented at the 10th Computer Vision Winter Workshop, Feb. 2005.
47. T. Walter and J.-C. Klein, “Automatic Analysis of Color Fundus
Photographs and Its Application to the Diagnosis of Diabetic Retinopathy
BT - Handbook of Biomedical Image Analysis: Volume II: Segmentation
Models Part B,” J. S. Suri, D. L. Wilson, and S. Laxminarayan, Eds.
Boston, MA: Springer US, 2005, pp. 315–368.
48. T. Walter, “Application de la Morphologie Mathématique au diagnostic de
la Rétinopathie Diabétique à partir d’images couleur,” PhD thesis, Mines
ParisTech, Fontainebleau, France, 2003.
49. T. Walter and J.-C. Klein, “Automatic Detection of Microaneurysms in
Color Fundus Images of the Human Retina by Means of the Bounding Box
Closing,” 2002, vol. 2526, pp. 210–220.
50. T. Walter and J.-C. Klein, “A Computational Approach to Diagnosis of
Diabetic Retinopathy,” in *6th Conference on Systemics, Cybernetics and
Informatics (SCI)*, 2002, pp. 521–526.
51. T. Walter, J.-C. Klein, P. Massin, and A. Erginay, “A Contribution of
Image Processing to the Diagnosis of Diabetic Retinopathy — Detection of
Exudates in Color Fundus Images of the Human Retina,” *IEEE Transactions
on Medical Imaging*, vol. 21, no. 10, pp. 1236–1243, 2002, doi:
[10.1109/TMI.2002.806290](https://doi.org/10.1109/TMI.2002.806290).
52. T. Walter and J.-C. Klein, “Segmentation of Color Fundus Images of the
Human Retina : Detection of the Optic Disc and the Vascular Tree Using
Morphological Techniques,” 2001, vol. 2199, pp. 282–287, doi:
[10.1007/3-540-45497-7\_43](https://doi.org/10.1007/3-540-45497-7_43).
53. T. Walter, J.-C. Klein, P. Massin, and F. Zana, “Automatic segmentation
and registration of retinal fluorescein angiographies - Application to
diabetic retinopathy,” presented at the First International Workshop on
Computer Assisted Fundus Image Analysis (CAFIA), May 2000.
54. T. Walter, “Gewinnung von Merkmalen transitorisch evozierter
otoakustischer Emissionen,” Diploma Thesis, Saarland university,
Saarbrücken, Germany, 1999.
{: reversed="reversed"}
