---
layout: page
title: Publications
permalink: /publications/
---

<div id="refs" class="references csl-bib-body" role="doc-bibliography">
<div id="ref-Chouaib2020" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[1] </div><div class="csl-right-inline">R. Chouaib <em>et al.</em>, <span>“A <span>Dual Protein</span>-<span class="nocase">mRNA Localization Screen Reveals Compartmentalized Translation</span> and <span>Widespread Co</span>-translational <span>RNA Targeting</span>,”</span> <em>Developmental Cell</em>, vol. 54, no. 6, pp. 773–791.e5, Sep. 2020, doi: <a href="https://doi.org/10.1016/j.devcel.2020.07.010">10.1016/j.devcel.2020.07.010</a>.</div>
</div>
<div id="ref-Walter2020" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[2] </div><div class="csl-right-inline">T. Walter, <span>“<span class="nocase">Bioimage Informatics for Phenomics</span>,”</span> Habilitation à diriger des recherches, <span>Sorbonne Universit<span>é</span></span>, 2020.</div>
</div>
<div id="ref-J.Boyd2020a" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[3] </div><div class="csl-right-inline">J. Boyd, Z. Gouveia, F. Perez, and T. Walter, <span>“Experimentally-<span>Generated Ground Truth</span> for <span>Detecting Cell Types</span> in an <span>Image</span>-<span>Based Immunotherapy Screen</span>,”</span> in <em>2020 <span>IEEE</span> 17th <span>International Symposium</span> on <span>Biomedical Imaging</span> (<span>ISBI</span>)</em>, Apr. 2020, pp. 886–890, doi: <a href="https://doi.org/10.1109/ISBI45749.2020.9098696">10.1109/ISBI45749.2020.9098696</a>.</div>
</div>
<div id="ref-Boyd2019" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[4] </div><div class="csl-right-inline">J. C. Boyd, A. Pinheiro, E. D. Nery, F. Reyal, and T. Walter, <span>“Domain-invariant features for mechanism of action prediction in a multi-cell-line drug screen,”</span> <em>Bioinformatics</em>, Oct. 2019, doi: <a href="https://doi.org/10.1093/bioinformatics/btz774">10.1093/bioinformatics/btz774</a>.</div>
</div>
<div id="ref-Durand2019" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[5] </div><div class="csl-right-inline">M. Durand <em>et al.</em>, <span>“Human lymphoid organ <span class="nocase">cDC2</span> and macrophages play complementary roles in <span>T</span> follicular helper responses,”</span> <em>The Journal of Experimental Medicine</em>, vol. 216, no. 7, pp. 1561 LP–1581, Jul. 2019, doi: <a href="https://doi.org/10.1084/jem.20181994">10.1084/jem.20181994</a>.</div>
</div>
<div id="ref-Dubois2019" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[6] </div><div class="csl-right-inline">R. Dubois <em>et al.</em>, <span>“A deep learning approach to identify <span class="nocase">mRNA</span> localization patterns,”</span> in <em><span>IEEE</span> 16th <span>International Symposium</span> on <span>Biomedical Imaging</span> (<span>ISBI</span> 2019)</em>, 2019, pp. 1386–1390, doi: <a href="https://doi.org/10.1109/ISBI.2019.8759235">10.1109/ISBI.2019.8759235</a>.</div>
</div>
<div id="ref-Kuperstein2019" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[7] </div><div class="csl-right-inline">I. Kuperstein and E. Barillot, Eds., <em>Computational <span>Systems Biology Approaches</span> in <span>Cancer Research</span></em>. <span>Chapman and Hall/CRC</span>, 2019.</div>
</div>
<div id="ref-Naylor2019a" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[8] </div><div class="csl-right-inline">P. Naylor, J. Boyd, M. Laé, F. Reyal, and T. Walter, <span>“Predicting <span>Residual Cancer Burden In A Triple Negative Breast Cancer Cohort</span>,”</span> in <em>2019 <span>IEEE</span> 16th <span>International Symposium</span> on <span>Biomedical Imaging</span> (<span>ISBI</span> 2019)</em>, 2019, pp. 933–937, doi: <a href="https://doi.org/10.1109/ISBI.2019.8759205">10.1109/ISBI.2019.8759205</a>.</div>
</div>
<div id="ref-Naylor2019" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[9] </div><div class="csl-right-inline">P. Naylor, M. La, F. Reyal, and T. Walter, <span>“Segmentation of <span>Nuclei</span> in <span>Histopathology Images</span> by deep regression of the distance map,”</span> <em>IEEE Transactions on Medical Imaging</em>, vol. 38, no. 2, pp. 448–459, 2019, doi: <a href="https://doi.org/10.1109/TMI.2018.2865709">10.1109/TMI.2018.2865709</a>.</div>
</div>
<div id="ref-Chouaib2018" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[10] </div><div class="csl-right-inline">R. Chouaib <em>et al.</em>, <span>“A <span>Localization Screen Reveals Translation Factories</span> and <span>Widespread Co</span>-<span>Translational Protein Targeting</span>,”</span> <span>Social Science Research Network</span>, <span>Rochester, NY</span>, SSRN Scholarly Paper ID 3300043, Dec. 2018. Accessed: Feb. 23, 2020. [Online]. Available: <a href="https://papers.ssrn.com/abstract=3300043">https://papers.ssrn.com/abstract=3300043</a>.</div>
</div>
<div id="ref-Boyd2018" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[11] </div><div class="csl-right-inline">J. Boyd, A. Pinhiero, E. D. Nery, F. Reyal, and T. Walter, <span>“Analysing double-strand breaks in cultured cells for drug screening applications by causal inference,”</span> in <em>International <span>Symposium</span> on <span>Biomedical Imaging</span> (<span>ISBI</span>): <span>From Nano</span> to <span>Macro</span></em>, 2018, vol. 2018–April, pp. 445–448, doi: <a href="https://doi.org/10.1109/ISBI.2018.8363612">10.1109/ISBI.2018.8363612</a>.</div>
</div>
<div id="ref-Samacoits2018" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[12] </div><div class="csl-right-inline">A. Samacoits <em>et al.</em>, <span>“A computational framework to study sub-cellular <span>RNA</span> localization,”</span> <em>Nature Communications</em>, vol. 9, no. 1, p. 4584, 2018, doi: <a href="https://doi.org/10.1038/s41467-018-06868-w">10.1038/s41467-018-06868-w</a>.</div>
</div>
<div id="ref-Bernard2017" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[13] </div><div class="csl-right-inline">E. Bernard, Y. Jiao, E. Scornet, V. Stoven, T. Walter, and J.-P. Vert, <span>“Kernel multitask regression for toxicogenetics,”</span> <em>Molecular Informatics</em>, vol. 36, Jan. 2017, [Online]. Available: <a href="http://biorxiv.org/content/early/2017/08/01/171298.abstract">http://biorxiv.org/content/early/2017/08/01/171298.abstract</a>.</div>
</div>
<div id="ref-Naylor2017" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[14] </div><div class="csl-right-inline">P. Naylor, M. Lae, F. Reyal, and T. Walter, <span>“Nuclei <span>Segmentation</span> in <span>Histopathology Images Using Deep Neural Networks</span>,”</span> <em>2017 IEEE 14th International Symposium on Biomedical Imaging (ISBI 2017)</em>, pp. IEEE; EMB; IEEE Signal Proc Soc, 2017, doi: <a href="https://doi.org/10.1109/ISBI.2017.7950669">10.1109/ISBI.2017.7950669</a>.</div>
</div>
<div id="ref-Isokane2016" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[15] </div><div class="csl-right-inline">M. Isokane <em>et al.</em>, <span>“<span>ARHGEF17</span> is an essential spindle assembly checkpoint factor that targets <span>Mps1</span> to kinetochores,”</span> <em>The Journal of Cell Biology</em>, vol. 212, no. 6, pp. 647–659, Mar. 2016, doi: <a href="https://doi.org/10.1083/jcb.201408089">10.1083/jcb.201408089</a>.</div>
</div>
<div id="ref-Machairas2016" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[16] </div><div class="csl-right-inline">V. Machairas, T. Baldeweck, T. Walter, and Etienne Décencière, <span>“New <span>General Features Based</span> on <span>Superpixels</span> for image <span>Segmentation Learning</span>,”</span> in <em>International <span>Symposium</span> on <span>Biomedical Imaging</span> (<span>ISBI</span>): <span>From Nano</span> to <span>Macro</span></em>, 2016, pp. (accepted for publication).</div>
</div>
<div id="ref-Tsanov2016" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[17] </div><div class="csl-right-inline">N. Tsanov <em>et al.</em>, <span>“<span>SmiFISH</span> and <span>FISH</span>-quant - <span>A</span> flexible single <span>RNA</span> detection approach with super-resolution capability,”</span> <em>Nucleic Acids Research</em>, vol. 44, no. 22, 2016, doi: <a href="https://doi.org/10.1093/nar/gkw784">10.1093/nar/gkw784</a>.</div>
</div>
<div id="ref-Eduati2015" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[18] </div><div class="csl-right-inline">F. Eduati <em>et al.</em>, <span>“Prediction of human population responses to toxic compounds by a collaborative competition,”</span> <em>Nature Biotechnology</em>, vol. 33, p. 933, Aug. 2015, [Online]. Available: <a href="https://doi.org/10.1038/nbt.3299">https://doi.org/10.1038/nbt.3299</a>.</div>
</div>
<div id="ref-Machairas2015b" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[19] </div><div class="csl-right-inline">V. Machairas, E. Decencière, and T. Walter, <span>“Spatial <span>Repulsion Between Markers Improves Watershed Performance</span>,”</span> 2015.</div>
</div>
<div id="ref-Machairas2015a" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[20] </div><div class="csl-right-inline">V. Machairas, E. Decencière, and T. Walter, <span>“Spatial <span>Repulsion Between Markers Improves Watershed Performance BT</span> - <span>Mathematical Morphology</span> and <span>Its Applications</span> to <span>Signal</span> and <span>Image Processing</span>,”</span> 2015, pp. 194–202.</div>
</div>
<div id="ref-Machairas2015" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[21] </div><div class="csl-right-inline">V. Machairas, M. Faessel, D. Cárdenas-peña, T. Chabardes, T. Walter, and E. Decencière, <span>“Waterpixels,”</span> <em>IEEE Transactions on Image Processing</em>, vol. 24, no. 11, pp. 3707–3716, 2015.</div>
</div>
<div id="ref-SchoenauerSebag2015" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[22] </div><div class="csl-right-inline">A. Schoenauer Sebag, S. Plancade, C. Raulet-Tomkiewicz, R. Barouki, J.-P. Vert, and T. Walter, <span>“A generic methodological framework for studying single cell motility in high-throughput time-lapse data,”</span> <em>Bioinformatics</em>, vol. 31, no. 12, pp. i320–i328, 2015, doi: <a href="https://doi.org/10.1093/bioinformatics/btv225">10.1093/bioinformatics/btv225</a>.</div>
</div>
<div id="ref-SchoenauerSebag2015a" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[23] </div><div class="csl-right-inline">A. Schoenauer Sebag, S. Plancade, C. Raulet-Tomkiewicz, R. Barouki, J.-P. Vert, and T. Walter, <span>“Infering an ontology of single cell motions from high-throughput microscopy data,”</span> in <em>Proceedings of the 12th <span>IEEE International Symposium</span> on <span>Biomedical Imaging</span> (<span>ISBI</span>): <span>From</span> nano to macro</em>, 2015, pp. 160–163.</div>
</div>
<div id="ref-Tegha-Dunghu2014" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[24] </div><div class="csl-right-inline">J. Tegha-Dunghu <em>et al.</em>, <span>“<span>MAP1S</span> controls microtubule stability throughout the cell cycle in human cells,”</span> <em>Journal of Cell Science</em>, vol. 127, no. 23, pp. 5007 LP–5013, Dec. 2014, doi: <a href="https://doi.org/10.1242/jcs.136457">10.1242/jcs.136457</a>.</div>
</div>
<div id="ref-Costello2014" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[25] </div><div class="csl-right-inline">J. C. Costello <em>et al.</em>, <span>“A community effort to assess and improve drug sensitivity prediction algorithms.”</span> <em>Nature biotechnology</em>, vol. 32, no. 12, pp. 1–103, 2014, doi: <a href="https://doi.org/10.1038/nbt.2877">10.1038/nbt.2877</a>.</div>
</div>
<div id="ref-Graml2014" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[26] </div><div class="csl-right-inline">V. Graml <em>et al.</em>, <span>“A <span>Genomic Multiprocess Survey</span> of <span>Machineries</span> that control and <span>Link Cell Shape</span>, <span>Microtubule Organization</span>, and <span>Cell</span>-<span>Cycle Progression</span>,”</span> <em>Developmental Cell</em>, vol. 31, no. 2, pp. 227–239, 2014, doi: <a href="https://doi.org/10.1016/j.devcel.2014.09.005">10.1016/j.devcel.2014.09.005</a>.</div>
</div>
<div id="ref-Heriche2014" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[27] </div><div class="csl-right-inline">J.-K. Hériché, J. G. Lees, I. Morilla, T. Walter, and B. Petrova, <span>“Integration of biological data by kernels on graph nodes allows prediction of new genes involved in mitotic chromosome condensation,”</span> <em>Molecular Biology of the cell</em>, vol. 25, pp. 2522–2536, 2014, doi: <a href="https://doi.org/10.1091/mbc.E13-04-0221">10.1091/mbc.E13-04-0221</a>.</div>
</div>
<div id="ref-Machairas2014" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[28] </div><div class="csl-right-inline">V. Machairas, E. Decencière, and T. Walter, <span>“Waterpixels: Superpixels based on the watershed transformation,”</span> in <em>International <span>Conference</span> on <span>Image Processing</span> (<span>ICIP</span>)</em>, 2014, pp. 4343–4347.</div>
</div>
<div id="ref-Veta2014" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[29] </div><div class="csl-right-inline">M. Veta <em>et al.</em>, <span>“Assessment of algorithms for mitosis detection in breast cancer histopathology images,”</span> <em>Medical Image Analysis</em>, pp. 1–23, 2014.</div>
</div>
<div id="ref-Pau2013" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[30] </div><div class="csl-right-inline">G. Pau, T. Walter, B. Neumann, J. Hériché, J. Ellenberg, and W. Huber, <span>“Dynamical modelling of phenotypes in a genome-wide <span>RNAi</span> live-cell imaging assay,”</span> <em>BMC bioinformatics</em>, vol. 14, no. 308, pp. 1–10, 2013, doi: <a href="https://doi.org/10.1186/1471-2105-14-308">10.1186/1471-2105-14-308</a>.</div>
</div>
<div id="ref-Mall2012" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[31] </div><div class="csl-right-inline">M. Mall <em>et al.</em>, <span>“Mitotic lamin disassembly is triggered by lipid-mediated signaling.”</span> <em>The Journal of cell biology</em>, vol. 198, no. 6, pp. 981–90, Sep. 2012, doi: <a href="https://doi.org/10.1083/jcb.201205103">10.1083/jcb.201205103</a>.</div>
</div>
<div id="ref-Conrad2011" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[32] </div><div class="csl-right-inline">C. Conrad <em>et al.</em>, <span>“Micropilot: Automation of fluorescence microscopy-based imaging for systems biology.”</span> <em>Nature methods</em>, vol. 8, no. 3, pp. 246–9, Mar. 2011, doi: <a href="https://doi.org/10.1038/nmeth.1558">10.1038/nmeth.1558</a>.</div>
</div>
<div id="ref-Held2010" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[33] </div><div class="csl-right-inline">M. Held <em>et al.</em>, <span>“<span>CellCognition</span>: Time-resolved phenotype annotation in high-throughput live cell imaging.”</span> <em>Nature methods</em>, vol. 7, no. 9, pp. 747–54, Sep. 2010, doi: <a href="https://doi.org/10.1038/nmeth.1486">10.1038/nmeth.1486</a>.</div>
</div>
<div id="ref-Dupas2010" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[34] </div><div class="csl-right-inline">B. Dupas <em>et al.</em>, <span>“Evaluation of automated fundus photograph analysis algorithms for detecting microaneurysms, haemorrhages and exudates, and of a computer-assisted diagnostic system for grading diabetic retinopathy.”</span> <em>Diabetes &amp; metabolism</em>, vol. 36, no. 3, pp. 213–220, Jun. 2010, doi: <a href="https://doi.org/10.1016/j.diabet.2010.01.002">10.1016/j.diabet.2010.01.002</a>.</div>
</div>
<div id="ref-Neumann2010" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[35] </div><div class="csl-right-inline">T. Walter <em>et al.</em>, <span>“Phenotypic profiling of the human genome by time-lapse microscopy reveals cell division genes.”</span> <em>Nature</em>, vol. 464, no. 7289, pp. 721–7, Apr. 2010, doi: <a href="https://doi.org/10.1038/nature08869">10.1038/nature08869</a>.</div>
</div>
<div id="ref-Walter2010" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[36] </div><div class="csl-right-inline">T. Walter <em>et al.</em>, <span>“Automatic identification and clustering of chromosome phenotypes in a genome wide <span>RNAi</span> screen by time-lapse imaging.”</span> <em>Journal of structural biology</em>, vol. 170, no. 1, pp. 1–9, Apr. 2010, doi: <a href="https://doi.org/10.1016/j.jsb.2009.10.004">10.1016/j.jsb.2009.10.004</a>.</div>
</div>
<div id="ref-Donoghue2010" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[37] </div><div class="csl-right-inline">S. I. O. Donoghue <em>et al.</em>, <span>“Visualizing biological data — now and in the future,”</span> <em>Nature Methods</em>, vol. 7, pp. S2–S4, 2010, doi: <a href="https://doi.org/10.1038/nmeth0310-S2">10.1038/nmeth0310-S2</a>.</div>
</div>
<div id="ref-Terjung2010" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[38] </div><div class="csl-right-inline">S. Terjung, T. Walter, A. Seitz, B. Neumann, R. Pepperkok, and J. Ellenberg, <span>“High-throughput microscopy using live mammalian cells.”</span> in <em>Live <span>Cell Imaging</span>: <span>A Laboratory Manual</span></em>, R. D. Goldman, J. R. Swedlow, and D. L. Spector, Eds. <span>Cold Spring Harbor Laboratory Press, Cold Spring Harbor</span>, 2010.</div>
</div>
<div id="ref-Walter2010a" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[39] </div><div class="csl-right-inline">T. Walter <em>et al.</em>, <span>“Visualization of image data from cells to organisms,”</span> <em>Nature Methods</em>, vol. 7, no. 3, pp. S26–S41, 2010, doi: <a href="https://doi.org/10.1038/NmEtH.1431">10.1038/NmEtH.1431</a>.</div>
</div>
<div id="ref-Tegha-Dunghu2008" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[40] </div><div class="csl-right-inline">J. Tegha-Dunghu <em>et al.</em>, <span>“<span>EML3</span> is a nuclear microtubule-binding protein required for the correct alignment of chromosomes in metaphase,”</span> <em>Journal of Cell Science</em>, vol. 121, no. 10, pp. 1718 LP–1726, May 2008, doi: <a href="https://doi.org/10.1242/jcs.019174">10.1242/jcs.019174</a>.</div>
</div>
<div id="ref-Walter2008" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[41] </div><div class="csl-right-inline">T. Walter <em>et al.</em>, <span>“A genome wide <span>RNAi</span> screen by time lapse microscopy in order to identify mitotic genes - <span>Computational</span> aspects and challenges,”</span> <em>2008 5th IEEE International Symposium on Biomedical Imaging: From Nano to Macro, Proceedings, ISBI</em>, pp. 328–331, 2008, doi: <a href="https://doi.org/10.1109/ISBI.2008.4540999">10.1109/ISBI.2008.4540999</a>.</div>
</div>
<div id="ref-Denion2007" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[42] </div><div class="csl-right-inline">E. Denion, J.-R. Ordonez, J.-C. Klein, A. Glacet-Bernard, T. Walter, and G. Caputo, <span>“Redistribution of the neurosensory retina in inferior limited macular translocation: An evaluation using image registration.”</span> <em>Graefe’s archive for clinical and experimental ophthalmology = Albrecht von Graefes Archiv fur klinische und experimentelle Ophthalmologie</em>, vol. 245, no. 3, pp. 437–442, Mar. 2007, doi: <a href="https://doi.org/10.1007/s00417-006-0408-1">10.1007/s00417-006-0408-1</a>.</div>
</div>
<div id="ref-Sihn2007" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[43] </div><div class="csl-right-inline">G. Sihn <em>et al.</em>, <span>“Anti-angiogenic properties of myo-inositol trispyrophosphate in ovo and growth reduction of implanted glioma.”</span> <em>FEBS letters</em>, vol. 581, no. 5, pp. 962–6, Mar. 2007, doi: <a href="https://doi.org/10.1016/j.febslet.2007.01.079">10.1016/j.febslet.2007.01.079</a>.</div>
</div>
<div id="ref-Erfle2007" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[44] </div><div class="csl-right-inline">H. Erfle <em>et al.</em>, <span>“Reverse transfection on cell arrays for high content screening microscopy,”</span> <em>Nature Protocols</em>, vol. 2, no. 2, pp. 392–399, Feb. 2007, doi: <a href="https://doi.org/10.1038/nprot.2006.483">10.1038/nprot.2006.483</a>.</div>
</div>
<div id="ref-Walter2007" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[45] </div><div class="csl-right-inline">T. Walter, P. Massin, A. Erginay, R. Ordonez, C. Jeulin, and J.-C. Klein, <span>“Automatic detection of microaneurysms in color fundus images.”</span> <em>Medical Image Analysis</em>, vol. 11, no. 6, pp. 555–66, 2007.</div>
</div>
<div id="ref-Walter2005a" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[46] </div><div class="csl-right-inline">T. Walter, R. Ordonez, and J.-C. Klein, <span>“A morphological approach for skeleton filtering with reconstruction of the relevant branches,”</span> presented at the 10th <span>Computer Vision Winter Workshop</span>, Feb. 2005.</div>
</div>
<div id="ref-Walter2005" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[47] </div><div class="csl-right-inline">T. Walter and J.-C. Klein, <span>“Automatic <span>Analysis</span> of <span>Color Fundus Photographs</span> and <span>Its Application</span> to the <span>Diagnosis</span> of <span>Diabetic Retinopathy BT</span> - <span>Handbook</span> of <span>Biomedical Image Analysis</span>: <span>Volume II</span>: <span>Segmentation Models Part B</span>,”</span> J. S. Suri, D. L. Wilson, and S. Laxminarayan, Eds. <span>Boston, MA</span>: <span>Springer US</span>, 2005, pp. 315–368.</div>
</div>
<div id="ref-Walter2003" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[48] </div><div class="csl-right-inline">T. Walter, <span>“Application de la Morphologie Mathématique au diagnostic de la Rétinopathie Diabétique à partir d’images couleur,”</span> PhD thesis, <span>Mines ParisTech</span>, <span>Fontainebleau, France</span>, 2003.</div>
</div>
<div id="ref-Walter2002b" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[49] </div><div class="csl-right-inline">T. Walter and J.-C. Klein, <span>“Automatic <span>Detection</span> of <span>Microaneurysms</span> in <span>Color Fundus Images</span> of the <span>Human Retina</span> by <span>Means</span> of the <span>Bounding Box Closing</span>,”</span> 2002, vol. 2526, pp. 210–220.</div>
</div>
<div id="ref-Walter2002c" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[50] </div><div class="csl-right-inline">T. Walter and J.-C. Klein, <span>“A <span>Computational Approach</span> to <span>Diagnosis</span> of <span>Diabetic Retinopathy</span>,”</span> in <em>6th <span>Conference</span> on <span>Systemics</span>, <span>Cybernetics</span> and <span>Informatics</span> (<span>SCI</span>)</em>, 2002, pp. 521–526.</div>
</div>
<div id="ref-Walter2002" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[51] </div><div class="csl-right-inline">T. Walter, J.-C. Klein, P. Massin, and A. Erginay, <span>“A <span>Contribution</span> of <span>Image Processing</span> to the <span>Diagnosis</span> of <span>Diabetic Retinopathy</span> — <span>Detection</span> of <span>Exudates</span> in <span>Color Fundus Images</span> of the <span>Human Retina</span>,”</span> <em>IEEE Transactions on Medical Imaging</em>, vol. 21, no. 10, pp. 1236–1243, 2002, doi: <a href="https://doi.org/10.1109/TMI.2002.806290">10.1109/TMI.2002.806290</a>.</div>
</div>
<div id="ref-Walter2001" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[52] </div><div class="csl-right-inline">T. Walter and J.-C. Klein, <span>“Segmentation of <span>Color Fundus Images</span> of the <span>Human Retina</span> : <span>Detection</span> of the <span>Optic Disc</span> and the <span>Vascular Tree Using Morphological Techniques</span>,”</span> 2001, vol. 2199, pp. 282–287, doi: <a href="https://doi.org/10.1007/3-540-45497-7_43">10.1007/3-540-45497-7_43</a>.</div>
</div>
<div id="ref-Walter2000" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[53] </div><div class="csl-right-inline">T. Walter, J.-C. Klein, P. Massin, and F. Zana, <span>“Automatic segmentation and registration of retinal fluorescein angiographies - <span>Application</span> to diabetic retinopathy,”</span> presented at the First <span>International Workshop</span> on <span>Computer Assisted Fundus Image Analysis</span> (<span>CAFIA</span>), May 2000.</div>
</div>
<div id="ref-Walter1999" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">[54] </div><div class="csl-right-inline">T. Walter, <span>“Gewinnung von Merkmalen transitorisch evozierter otoakustischer Emissionen,”</span> Diploma Thesis, <span>Saarland university</span>, <span>Saarbrücken, Germany</span>, 1999.</div>
</div>
</div>

